win32port/win32helpers/gettimeofday.c

```diff
@@ -1,5 +1,5 @@
 #include <time.h>
-#include <windows.h> //I've omitted context line

```

Comment:
#include "gettimeofday.h" is required to avoid the following compile error:

w:/libwebsockets/win32port/win32helpers/gettimeofday.c:22:45: warning: 'struct timezone' declared inside parameter list [enabled by default]
 int gettimeofday(struct timeval *tv, struct timezone *tz)
                                             ^
w:/libwebsockets/win32port/win32helpers/gettimeofday.c:22:45: warning: its scope is only this definition or declaration, which is probably not what you want [enabled by default]
w:/libwebsockets/win32port/win32helpers/gettimeofday.c: In function 'gettimeofday':
w:/libwebsockets/win32port/win32helpers/gettimeofday.c:47:5: error: dereferencing pointer to incomplete type
   tz->tz_minuteswest = _timezone / 60;
     ^
w:/libwebsockets/win32port/win32helpers/gettimeofday.c:48:5: error: dereferencing pointer to incomplete type
   tz->tz_dsttime = _daylight;
     ^
make[2]: *** [CMakeFiles/test-client.dir/win32port/win32helpers/gettimeofday.c.obj] Error 1
make[1]: *** [CMakeFiles/test-client.dir/all] Error 2
make: *** [all] Error 2
[ 19%] Built target zlib_internal
[ 45%] Built target websockets

However, it is no problem to keep the #include <windows.h>. I'll commit that and check what the builds say.